platform/src/main/java/org/stellar/anchor/platform/config/FireblocksConfig.java

```diff
@@ -39,6 +46,36 @@ public void setRetryConfig(RetryConfig retryConfig) {
     this.retryConfig = retryConfig;
   }
 
+  public void setAssetMappings(String assetMappings) {
+    if (StringUtils.isEmpty(assetMappings)) {
+      this.assetMappings = Collections.emptyMap();
+    } else {
+      this.assetMappings =
+          Arrays.stream(assetMappings.split(StringUtils.LF))
+              .collect(
+                  Collectors.toMap(
+                      mapping -> mapping.substring(mapping.indexOf(' ') + 1),
+                      mapping -> mapping.substring(0, mapping.indexOf(' '))));
+    }
+  }
+
+  /**
+   * Get Fireblocks asset code by Stellar asset code
+   *
+   * @return Fireblocks asset code or null if no mapping found
+   */
+  public String getFireblocksAssetCode(String stellarAssetCode) {
+    if (assetMappings.containsKey(stellarAssetCode)) {
+      return assetMappings.get(stellarAssetCode);
+    }
+
+    Log.warnF(
+        "Unable to find Fireblocks asset code by Stellar asset code [%s]."
```

Comment:
I am not sure whether it works only with {} or with both {} and %s for params

Reply:
Maybe it is better to throw exception? What do you think? Fireblocks will return it's error anyway. But we can discuss it